travel_journal/src/components/About.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faLocationDot } from "@fortawesome/free-solid-svg-icons"
+
+export default function About(props) {
```

Comment:
Two options:

1. you can destruct it: `About({item})`
and then use: `item.page`

2.  you can destruct it: `About({item: { page, name, startDate...}})`
and then use: `page`